src/services/item/plugins/invitation/index.ts

```diff
@@ -192,6 +210,52 @@ const plugin: FastifyPluginAsync = async (fastify) => {
       reply.status(StatusCodes.NO_CONTENT);
     },
   );
+
+  fastify.post<{ Params: { itemId: string } }>(
+    '/:itemId/enroll',
```

Comment:
I'm just using this branch for updating my front for now, but should this be in membership instead? or item login? It was not trivial for me that it is in invitation.